lib/breathe_client.rb

```diff
@@ -85,14 +95,19 @@ def training_events(person:, after:)
           start_date = training[:start_on]&.to_date
 
           if start_date.nil?
-            puts "[DEBUG] Skipping training with nil start_date for #{person.breathe_id}"
+            puts "[DEBUG] #{person.label}: Skipping training with nil start_date"
             next
           end
 
           end_date = training[:end_on]&.to_date
```

Comment:
could we move this next to start_date so we have the paired logic together, then all the error cases together?